hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1297,29 +1374,55 @@
       tests: []
     }
 
-    ////////////////////////////////////////////////
-    // Memory & Controllers                       //
-    // ROM, RAM, FLASH, FLASH_CTRL, OTP, OTP_CTRL //
-    ////////////////////////////////////////////////
+    /////////////////////////////////////////////////////
+    // Memory & Controllers                            //
+    // ROM_CTRL, RAM, FLASH, FLASH_CTRL, OTP, OTP_CTRL //
+    /////////////////////////////////////////////////////
 
-    // ROM (pre-verified IP) integration tests:
-    // TODO: Not sure if this will really be a pre-verified IP. If not, then more tests are needed
-    // at the chip level.
+    // ROM_CTRL (pre-verified IP) integration tests:
     {
       name: chip_rom_access
-      desc: '''Verify access to the rom.
+      desc: '''Verify that the CPU can access the rom contents.
 
-            Verify that the CPU can fetch instructions from the ROM. Nothing extra needs to be done
-            here - all SW tests do this anyway.
+            - Verify that the CPU can fetch instructions from the ROM.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_rom_security_features
-      desc: '''Verify ROM security / ECC features if available.
+      name: chip_rom_ctrl_ast_cfg
+      desc: '''Verify that the connectivity of AST cfg signals to the ROM ctrl.
```

Comment:
Nit: If you're rebasing anyway, there's a stray "that" here.

Reply:
Done.